ci/jenkins/test-vmc.sh

```diff
@@ -404,7 +404,19 @@ function garbage_collection() {
     echo "=== Auto cleanup starts ==="
     export KUBECONFIG=$KUBECONFIG_PATH
 
-    kubectl get namespace -l antrea-ci | awk '$3 ~ "[0-9][hd]" || $3 ~ "[6-9][0-9]m" || $3 ~ "1[0-9][0-9]m" && $2 ~ "Active" {print $1}' | while read cluster; do
+    kubectl get namespace -l antrea-ci | awk '$1 ~ "matrix" && $3 ~ "[0-9]d" || $3 ~ "[4-9]h" || $3 ~ "[3-9][0-9][0-9]m" || $3 ~ "2[4-9][0-9]m" && $2 ~ "Active" {print $1}' | while read cluster; do
```

Comment:
I think the regular expressions for matching dates are too complicated. Maybe let's use the following style to make it more readable:

```
kubectl get ns -o custom-columns=Name:.metadata.name,DATE:.metadata.creationTimestamp --no-headers=true | awk '{cmd="echo $(( $(date +%s) - $(date -d "$2" +%s) ))"; cmd | getline t ; print $1, t}' | awk '$1 ~ "matrix" && $2 > 14400 {print $1}' | while read cluster; do
    ...
done
```
Here 14400 is 4 Hours =14400 seconds. So this would be more flexible when we want to change the time length.

Reply:
Changed script style.